### PR TITLE
Upgrade environment on 2022-11-22

### PR DESCRIPTION
**Pull request by dhimmel**
Includes Python 3.11 & Pandoc 2.18

### CODE REVIEW

File: build/environment.yml [hunk 1]
```diff
@@ -4,34 +4,33 @@ channels:
...
   - cairo=1.16.0
   - cairocffi=1.2.0
-  - ghp-import=2.0.2
-  - jinja2=3.0.3
-  - jsonschema=4.2.1
-  - pandoc=2.17.0.1
+  - ghp-import=2.1.0
+  - jinja2=3.1.2
+  - jsonschema=4.17.0
+  - pandoc=2.19.2
```
Comment (dhimmel) [inline build/environment.yml:10]: I think the pandoc update from 2.17.0.1 to 2.19.2 might be causing this issue:

image

which is coming from the following rendered HTML in <head>:
```html
<!--
  | Manubot generated metadata rendered from header-includes-template.html.
  | Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  | --> ¶ <meta name="dc.format" content="text/html" /> ¶ <meta property="og:type" content="article" /> ¶ <meta name="dc.title" content="Manuscript Title" /> ¶ <meta name="citation_title" content="Manuscript Title" /> ¶ <meta property="og:title" content="Manuscript Title" /> ¶ <meta property="twitter:title" content="Manuscript Title" /> ¶ <meta name="dc.date" content="2022-11-22" /> ¶ <meta name="citation_publication_date" content="2022-11-22" /> ¶ <meta property="article:published_time" content="2022-11-22" /> ¶ <meta name="dc.modified" content="2022-11-22T23:45:00+00:00" /> ¶ <meta property="article:modified_time" content="2022-11-22T23:45:00+00:00" /> ¶ <meta name="dc.language" content="en-US" /> ¶ <meta name="citation_language" content="en-US" /> ¶ <meta name="dc.relation.ispartof" content="Manubot" /> ¶ <meta name="dc.publisher" content="Manubot" /> ¶ <meta name="citation_journal_title" content="Manubot" /> ¶ <meta name="citation_technical_report_institution" content="Manubot" /> ¶ <meta name="citation_author" content="John Doe" /> ¶ <meta name="citation_author_institution" content="Department of Something, University of Whatever" /> ¶ <meta name="citation_author_orcid" content="XXXX-XXXX-XXXX-XXXX" /> ¶ <meta name="twitter:creator" content="@johndoe" /> ¶ <meta name="citation_author" content="Jane Roe" /> ¶ <meta name="citation_author_institution" content="Department of Something, University of Whatever" /> ¶ <meta name="citation_author_institution" content="Department of Whatever, University of Something" /> ¶ <meta name="citation_author_orcid" content="XXXX-XXXX-XXXX-XXXX" /> ¶ <link rel="canonical" href="https://manubot.github.io/rootstock/" /> ¶ <meta property="og:url" content="https://manubot.github.io/rootstock/" /> ¶ <meta property="twitter:url" content="https://manubot.github.io/rootstock/" /> ¶ <meta name="citation_fulltext_html_url" content="https://manubot.github.io/rootstock/" /> ¶ <meta name="citation_pdf_url" content="https://manubot.github.io/rootstock/manuscript.pdf" /> ¶ <link rel="alternate" type="application/pdf" href="https://manubot.github.io/rootstock/manuscript.pdf" />
  | <link rel="alternate" type="text/html" href="https://manubot.github.io/rootstock/v/a13936a0d3c1a9d35976e41bae6a4511cfc5daa3/" /> ¶ <meta name="manubot_html_url_versioned" content="https://manubot.github.io/rootstock/v/a13936a0d3c1a9d35976e41bae6a4511cfc5daa3/" /> ¶ <meta name="manubot_pdf_url_versioned" content="https://manubot.github.io/rootstock/v/a13936a0d3c1a9d35976e41bae6a4511cfc5daa3/manuscript.pdf" /> ¶ <meta property="og:type" content="article" /> ¶ <meta property="twitter:card" content="summary_large_image" /> ¶ <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  | <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" /> ¶ <meta name="theme-color" content="#ad1457" /> ¶ <!-- end Manubot generated metadata -->
```

`output/manuscript.md` looks fine like with a YAML head like:

```yaml
header-includes: |-
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta property="og:type" content="article" />
```

Comment (dhimmel) [reply build/environment.yml:10]: Opened https://github.com/jgm/pandoc/issues/8449, we can use Pandoc 2.18 for now.

**Comment by dhimmel** [comment]
HTML output from https://github.com/manubot/rootstock/pull/481/commits/4423e5e683f1688052cf93f2d30b77e30ad775b6 looks good to me (besides a pubmed ID citation due to an intermittent NCBI API outage).

**Review by agitter — APPROVED** [review]
I quickly looked at the outputs from 7a459c7 and didn't see any problems besides the pubmed reference you already noted.

**Pull request merged**